### PR TITLE
crDroidOTA: update enchilada & fajita with new maintainer

### PR DESCRIPTION
**Pull request by Terminator-J**
Hi there, my name's Jordan Whiteley (Terminator_J on XDA, Telegram, and GitHub).
Max is passing the baton to me for maintaining crDroid 7 on OnePlus 6 & 6T,
and this has definitely been a learning experience just getting to this point.
Looking forward to keeping the best rom ever going on our devices, into
Android 11 and hopefully beyond!

### CODE REVIEW

**Comment by Terminator-J** [comment]
Hey there. I'm intentionally not including an actual zip file in the json, as we're not actually ready to build & push March yet (still having some folks in the Telegram channel test for bugs first). 
Sorry the changelog is so big, it's actually redoing the device tree "the right way" instead of trying to bolt on luk1337's unofficial lineage-18.1 patchset from the LOS gerrit with all of the cruft from the last half a year of me not knowing wtf I was doing messing with things, or how to cleanup with a `git rebase -i HEAD~somenum` before. It's better now, honest.

**Comment by Terminator-J** [comment]
I guess I should wait for someone else to review and press that pretty "Squash & merge button". 
MergeOwnPRs

**Review by gwolf2u — CHANGES_REQUESTED** [review]
--- Do not merge ---
Mandatory tags need to be addressed
Wait for fix

**Pull request closed without merging**
